agent/firewall.go

```diff
@@ -42,13 +42,17 @@ import (
 )
 
 const (
-	numberOfDefaultChains = 3 // INPUT,OUTPUT,FORWARD
+	numberOfDefaultChains = 4 // INPUT,OUTPUT,FORWARD_IN,FORWARD_OUT,TENANT_VECTOR
```

Comment:
If you want to keep this value (and not perform a -1 calculation where you use it) then call it the "maxChainIndex" (or something like that).

I think it's clearer in this case, though, to stick with "number of chains" and perform a -1 calculation where necessary. Every developer knows that "number of" and "index of" needs a small conversion, so there wouldn't be anything complicated or surprising about using it that way.